### ISSUE TITLE
Outdated plotly.js in R package

### ISSUE THREAD

**Issue by yankev** [ropensci/plotly#626]
Current version in the package is 1.10.1, while the latest version is 1.13.0

```
Session info ---------------------------------------------------------------------------------------------------------------------------------------
 setting  value                       
 version  R version 3.2.3 (2015-12-10)
 system   x86_64, darwin13.4.0        
 ui       RStudio (0.99.491)          
 language (EN)                        
 collate  en_CA.UTF-8                 
 tz       America/Toronto             
 date     2016-06-16                  

Packages -------------------------------------------------------------------------------------------------------------------------------------------
 package     * version date       source        
 assertthat    0.1     2013-12-06 CRAN (R 3.2.0)
 base64enc     0.1-3   2015-07-28 CRAN (R 3.2.0)
 colorspace    1.2-6   2015-03-11 CRAN (R 3.2.0)
 DBI           0.4-1   2016-05-08 CRAN (R 3.2.5)
 devtools      1.11.1  2016-04-21 CRAN (R 3.2.5)
 digest        0.6.9   2016-01-08 CRAN (R 3.2.3)
 dplyr         0.4.3   2015-09-01 CRAN (R 3.2.0)
 ggplot2     * 2.1.0   2016-03-01 CRAN (R 3.2.4)
 gridExtra     2.2.1   2016-02-29 CRAN (R 3.2.4)
 gtable        0.2.0   2016-02-26 CRAN (R 3.2.3)
 htmltools     0.3.5   2016-03-21 CRAN (R 3.2.4)
 htmlwidgets   0.6     2016-02-25 CRAN (R 3.2.3)
 httr          1.1.0   2016-01-28 CRAN (R 3.2.3)
 jsonlite      0.9.20  2016-05-10 CRAN (R 3.2.5)
 magrittr      1.5     2014-11-22 CRAN (R 3.2.0)
 memoise       1.0.0   2016-01-29 CRAN (R 3.2.3)
 munsell       0.4.3   2016-02-13 CRAN (R 3.2.3)
 plotly      * 3.6.0   2016-05-18 CRAN (R 3.2.5)
 plyr          1.8.3   2015-06-12 CRAN (R 3.2.0)
 R6            2.1.2   2016-01-26 CRAN (R 3.2.3)
 Rcpp          0.12.5  2016-05-14 CRAN (R 3.2.5)
 scales        0.4.0   2016-02-26 CRAN (R 3.2.3)
 tidyr         0.4.1   2016-02-05 CRAN (R 3.2.3)
 viridis       0.3.4   2016-03-12 CRAN (R 3.2.4)
 withr         1.0.1   2016-02-04 CRAN (R 3.2.3)
 yaml          2.1.13  2014-06-12 CRAN (R 3.2.0)
```

**Comment by timelyportfolio** [comment]
I can fix this afternoon.  I have been waiting for @cpsievert changes, but probably better to go ahead and get to the latest version.

**Comment by yankev** [comment]
I made a branch yesterday to work with the updated version locally, so I just put it up as a PR.

**Comment by cpsievert** [comment]
Closed via d2bf2a1eb56d824a091e2cc9afea09a5968dd050.
